client/src/app/app.module.ts

```diff
@@ -18,6 +18,7 @@ import { SharedModule } from '@shared/shared.module';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { EmployeePayrollListComponent } from './employee/employee-payroll-list/employee-payroll-list.component';
```

Comment:
You should be able to import this from the alias, like the other imports

Reply:
fixed